Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM eclipse-temurin:21-jre-alpine
+
+ARG USER=default
+ENV HOME=/home/$USER
+
+ENV TZ=Europe/Oslo
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# install sudo as root
+RUN apk add --update sudo
+RUN adduser -D $USER \
+        && echo "$USER ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/$USER \
+        && chmod 0440 /etc/sudoers.d/$USER
```

Comment:
Dette er kopiert fra [catalog-view-api](https://github.com/Informasjonsforvaltning/catalog-view-api), men jeg er litt usikker på om der er noe vi trenger. Er det noen spesiell grunn til at vi har det? Om ikke antar jeg det er sikrere om vi ikke kjører med sudotilgang

Reply:
Det er bare for å opprette og ta i bruken en bruker, er vel noe best-practice greier som sier at en ikke bør kjøre som root i container. Du kan fint bare la det ligge